VOHE-Note.bib

```diff
@@ -137,11 +137,27 @@ @misc{std:W3CProvDM
     url =          {http://www.w3.org/TR/prov-dm}
 }
 
-@misc{unbehaun2023prospects,
-      title={Prospects for combined analyses of hadronic emission from $\gamma$-ray sources in the Milky Way with CTA and KM3NeT},
-      author={T. Unbehaun, L. Mohrmann, et al.},
-      year={2023},
-      eprint={2309.03007},
-      archivePrefix={arXiv},
-      primaryClass={astro-ph.HE}
+@article{giunti2022,
+	author = {{Giunti, L.} and {Acero, F.} and {Khélifi, B.} and {Kosack, K.} and {Lemière, A.} and {Terrier, R.}},
+	title = {Constraining leptonic emission scenarios for the PeVatron candidate HESS J1702−420 with deep XMM-Newton observations⋆},
```

Comment:
I think the issue is here : there is a Unicode character : − (U+2212) to be replaced
Maybe remove the * at the end too

Reply:
Which is a typo, by the way...